.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "nextpnr-xilinx"]
 	path = nextpnr-xilinx
 	url = git@github.com:kintex-chatter/nextpnr-xilinx.git
+[submodule "prjxray"]
+	path = prjxray
+	url = git@github.com:kintex-chatter/prjxray.git
```

Comment:
Replace by upstream, i.e. `url = git@github.com:f4pga/prjxray`?

Reply:
prjxray in kintex-chatter is ahead of jrrk2:prjxray, which is not aligned with f4pga:prjxray, so I'm hesitant to make this change without comment from the maintainers of the kintex-chatter:prjxray repo.

Reply:
Got it. We can move this to a separate issue.